src/sections/footer.html

```diff
@@ -1,4 +1,37 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="lg:hidden text-darkgreen text-[0.938rem] md:text-[1rem] font-medium">
+    <div class="md:flex justify-between md:mb-[1.313rem]"> 
+        <div id="footer__address" class="mb-[1.313rem]">
+            Agentur Baumeister <br>
+            Böckhstraße 13 <br>
+            10967 Berlin
+        </div>
+        <div id="footer__contact" class="mb-[7rem] md:mb-0 md:text-right">
+            <span class="underline">hallo@agentur-baumeister.com</span> <br>
+            +49 (0)30 - 490 827 87
+        </div>
+    </div>
+    <div id="footer__links" class="flex justify-between mb-[1.625rem] md:mb-[2.313rem]">
+        <a href="#" target="_blank">Datenschutz</a>
+        <a href="#" target="_blank">Impressum</a>
+        <a href="#" target="_blank">LinkedIn</a>
+        <a href="#" target="_blank">Instagram</a>
+    </div>
+</div>
+
+<div class="hidden lg:flex justify-between text-darkgreen text-[clamp(1rem, 1.58vw, 1.5rem)] font-medium">
+    <div id="footer__address" class="mb-[1.313rem]">
+        Agentur Baumeister <br>
+        Böckhstraße 13 <br>
+        10967 Berlin
+    </div>
+    <div id="footer__contact" class="text-right">
+        <span class="underline">hallo@agentur-baumeister.com</span> <br>
+        +49 (0)30 - 490 827 87
+    </div>
```

Comment:
Like Kevin mentioned, you should put every line in their own elements. Use "p" tags so they stack on top of each other. 
Also, the text is left aligned in the Figma design.

```suggestion
    <div id="footer__contact" class="text-left">
        <p class="underline">hallo@agentur-baumeister.com</p> 
        <p>+49 (0)30 - 490 827 87</p>
    </div>
```